docs/general/server/media/media-segments.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
i am somewhat missing the step-by-step instructions how to enable it.

Reply:
something like this?

## Getting started

There's a few steps to follow if you want to use media segments.

1. First, you'll need one or multiple plugins that can provide media segments. Read the [plugins](#) section below to find out more.
2. Run the "Media segment scan" task in the dashboard to create segments immediately, this task also runs automatically in the background.
3. Enable actions for the different segment types, the way you do this differs per client but they are generally found in the playback settings of the client.